### DISCUSSION TITLE
Is there any Flutter project that uses this UI and supports mobile platforms?

### DISCUSSION THREAD

**Discussion by cskime** [nank1ro/flutter-shadcn-ui#171] [Q&A] [answered]
Hello. I just found out about this package, and I love it! I’m planning to use it for my projects. 
However, it seems like shadcn is primarily designed for the web, and I’m not sure if its UI is suitable for mobile apps as well. 
Is there any Flutter project that uses this UI and supports mobile platforms?

**Comment by nank1ro** [comment, marked as answer]
Yes this UI works fine even on mobile.
For example the ContextMenu component can be triggered with a long press instead of right click.
The Dialog adapts to the size and changes its aspect on mobile (tiny) devices and so on.

You can check the apps created with Flutter ShadcnUI from the docs
https://flutter-shadcn-ui.mariuti.com/
![image](https://github.com/user-attachments/assets/d0c7b0a5-c6fe-48ab-a5cc-90c5042c52c4)

**Reply by cskime** [reply]
oh, I didn't notice that. I'll check them. Thanks!
